### DISCUSSION TITLE
Possible update to support PS Vita?

### DISCUSSION THREAD

**Discussion by assassinhacker01** [cathery/sys-con#77] [Controller requests]
Sorry to be doing this really appreciate your work. I've been wondering for ages now if it would be possible to use my PS Vita as a consel and your recent work gives me so much hope. Please consider this an option for a future update.🙏

**Comment by Masamune3210** [comment]
The Vita doesnt output anything to usb controller wise.

**Comment by assassinhacker01** [comment]
It has an app called vitapad. You have to install the app on your phone and
laptop but you can use it as a controller for either as long as connected
to the same wifi.

**Comment by Masamune3210** [comment]
But that's not over USB, that's just funneling the inputs over wifi

**Comment by cathery** [comment]
This is a Nintendo Switch sysmodule. I don't have the knowledge or experience working on PS Vita, or a PS Vita in general. You'll need some kind of app that makes your PS Vita be recognized as a controller first.

**Comment by assassinhacker01** [comment]
> But that's not over USB, that's just funneling the inputs over wifi

https://www.cfwaifu.com/vitastick-vita-controller/

Here's your over usb output

**Comment by assassinhacker01** [comment]
> This is a Nintendo Switch sysmodule. I don't have the knowledge or experience working on PS Vita, or a PS Vita in general. You'll need some kind of app that makes your PS Vita be recognized as a controller first.

https://www.cfwaifu.com/vitastick-vita-controller/

Please have a look at this

**Comment by cathery** [comment]
Support for generic HID devices will be added when a way to support generic HID devices is found.

**Comment by assassinhacker01** [comment]
Thank you so much😁 much appreciated. Looking forward to your work.
